guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -61,10 +48,26 @@ For more information, see the Red Hat Knowledgebase solution https://access.redh
 [[upgrade_paths]]
 == Upgrade Paths
 
+ifdef::satellite[]
+Before you upgrade your {Project}, you have to upgrade the operating system on your {ProjectServer}s and {SmartProxies} to {RHEL} 8.
+endif::[]
+ifndef::satellite[]
+Before you upgrade your {Project}, you have to upgrade the operating system on your {ProjectServer}s and {SmartProxies} to Enterprise Linux 8.
+endif::[]
```

Comment:
I'm also unsure I like how you split up the "Upgrade Paths" section now, but don't have a good alternative suggestion right now. Let me ramble a bit, and maybe you can distill the right things from my ramblings? :sweat_smile: 

* As noted in the section below, the user needs to be on `{ProjectVersionPrevious}` to be able to upgrade to `{ProjectVersion}` -- we don't support skipping releases.
* Once they are on `{ProjectVersionPrevious}`, they can do the "normal" upgrade as described in `== Upgrading {ProjectName}` chapter (ch2 upstream, ch3 downstream)
* At this point, they are running the latest available version and are "done" with the version upgrade
* They *can* now continue with the OS upgrade, but don't have to -- this can happen at any later point (as long as it happens before they decide to upgrade to a major version that does not support OS upgrades anymore, i.e. Foreman 3.4)
* They *also can* perform minor updates (so Foreman 3.3.1, 3.3.2 etc, as in the very last chapter of the update guide), while still on EL7.

I am sure there will be users who decide to do both jumps "at once", and also those who will just update to the latest version and schedule the OS upgrade at a later point.

Reply:
The sentence will be valid for 6.12 

For 6.11 it is vice versa like @evgeni said

Reply:
Do they have to be on the latest .z release to be able to upgrade?